08:40:33 PM: DEBUG [conexion.py:23] Conexion exitosa: <connection object at 0x000001FF4727AE10; dsn: 'user=postgres password=xxx dbname=test host=127.0.0.1 port=5432', closed: 0>
08:40:33 PM: DEBUG [conexion.py:35] se abrio el cursor:<cursor object at 0x000001FF491244A0; closed: 0>
08:40:54 PM: DEBUG [conexion.py:23] Conexion exitosa: <connection object at 0x000001819461AE10; dsn: 'user=postgres password=xxx dbname=test host=127.0.0.1 port=5432', closed: 0>
08:40:54 PM: DEBUG [conexion.py:35] se abrio el cursor:<cursor object at 0x00000181964D6580; closed: 0>
08:41:31 PM: DEBUG [persona.py:50] 
            id: 1
            nombre: JUan
            last Name: asv
            email: Jasv@mail.com
        
08:41:31 PM: DEBUG [persona.py:53] 
            id: 1
            nombre: JUan
            last Name: asv
            email: None
        
08:41:31 PM: DEBUG [persona.py:56] 
            id: 1
            nombre: None
            last Name: None
            email: None
        
08:41:36 PM: DEBUG [conexion.py:23] Conexion exitosa: <connection object at 0x000002ACF9F2AE10; dsn: 'user=postgres password=xxx dbname=test host=127.0.0.1 port=5432', closed: 0>
08:41:36 PM: DEBUG [conexion.py:35] se abrio el cursor:<cursor object at 0x000002ACFBDC9580; closed: 0>
08:42:40 PM: DEBUG [conexion.py:23] Conexion exitosa: <connection object at 0x0000024C1E9FAE10; dsn: 'user=postgres password=xxx dbname=test host=127.0.0.1 port=5432', closed: 0>
08:42:40 PM: DEBUG [conexion.py:35] se abrio el cursor:<cursor object at 0x0000024C20D69580; closed: 0>
